src/test/scala/mesosphere/marathon/api/validation/AppDefinitionValidatorTest.scala

```diff
@@ -325,6 +325,30 @@ class AppDefinitionValidatorTest extends MarathonSpec with Matchers with GivenWh
     assert(validate(container).isFailure)
   }
 
+  test("persistent volume with container path '.' is invalid") {
+    val f = new Fixture
+    val container = f.validDockerContainer.copy(
+      volumes = Seq(f.validPersistentVolume.copy(containerPath = "."))
+    )
+    assert(validate(container).isFailure)
+  }
+
+  test("persistent volume with container path '..' is invalid") {
```

Comment:
the current validation would allow `"foo..bar"` so I would test that as well to make it obvious – or disallow dots in general?